specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/vmssNetworkInterface.json

```diff
@@ -147,7 +147,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.Compute/virtualMachineScaleSets/{virtualMachineScaleSetName}/virtualMachines/{virtualmachineIndex}/networkInterfaces/{networkInterfaceName}": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.Compute/virtualMachineScaleSets/{virtualMachineScaleSetName}/virtualMachines/{virtualMachineIndex}/networkInterfaces/{networkInterfaceName}": {
```

Comment:
same above,  please make sure all checks in the sematic validation pass.

Reply:
they're